cypress/fixtures/api/insights-results-aggregator/v2/rule/external.rules.rule|ERROR_KEY/clusters_detail.json

```diff
@@ -7,183 +7,206 @@
-        }
+        },
+        "impacted": "2022-06-13T22:00:00.000Z"
       },
       {
         "cluster": "bcbd9c24-bf3f-4141-9a6c-019f5404164e",
         "cluster_name": "",
         "last_checked_at": "2020-02-22T08:32:37.690Z",
         "meta": {
           "cluster_version": "4.17.9"
-        }
+        },
+        "impacted": "2022-06-26T22:00:00.000Z"
       },
       {
         "cluster": "685de527-ad9e-426c-a83f-07d797dd28e8",
         "cluster_name": "foobar cluster",
         "last_checked_at": "2022-05-22T08:32:37.690Z",
         "meta": {
           "cluster_version": ""
-        }
+        },
+        "impacted": "2022-06-22T22:00:00.000Z"
       },
       {
         "cluster": "4bc920df-7935-4882-a946-e1e735439620",
         "cluster_name": "",
         "last_checked_at": "2020-02-22T08:32:37.690Z",
         "meta": {
           "cluster_version": "3.3.5"
-        }
+        },
+        "impacted": "2022-06-11T22:00:00.000Z"
       },
       {
         "cluster": "41c30565-b4c9-49f2-a4ce-3277ad22b258",
         "cluster_name": "",
         "last_checked_at": "2020-02-22T08:32:37.690Z",
         "meta": {
           "cluster_version": "4.17.9"
-        }
+        },
+        "impacted": "2022-06-11T22:00:00.000Z"
       },
       {
         "cluster": "6810f59e-c37c-420a-ae52-c23ccaccffc9",
         "cluster_name": "custom cluster name 1",
         "last_checked_at": "2021-04-02T08:32:37.690Z",
         "meta": {
           "cluster_version": "4.11.16"
-        }
+        },
+        "impacted": "2022-06-09T22:00:00.000Z"
       },
       {
         "cluster": "5d5892d3-1f74-4ccf-22af-548dfc9767bb",
         "cluster_name": "",
         "last_checked_at": "2021-04-08T20:35:32.798Z",
         "meta": {
           "cluster_version": "4.8.2"
-        }
+        },
+        "impacted": "2022-06-28T22:00:00.000Z"
       },
       {
         "cluster": "444cf4e5-a983-4894-9852-cb8b441b7006",
         "cluster_name": "",
         "last_checked_at": "2021-04-08T20:35:32.798Z",
         "meta": {
           "cluster_version": ""
-        }
+        },
+        "impacted": "2022-06-28T22:00:00.000Z"
       },
       {
         "cluster": "00d404e4-9e83-46ee-9ddd-83c06fa65d67",
         "cluster_name": "",
         "last_checked_at": "2021-04-08T20:35:32.798Z",
         "meta": {
           "cluster_version": "4.9.5"
-        }
+        },
+        "impacted": "2022-06-01T22:00:00.000Z"
       },
       {
         "cluster": "e73ee8b2-f60f-4bb0-abc7-be3093debfe7",
         "cluster_name": "",
         "last_checked_at": "2021-04-08T20:35:32.798Z",
         "meta": {
           "cluster_version": "4.18.12"
-        }
+        },
+        "impacted": ""
       },
       {
         "cluster": "f7331e9a-2f59-484d-af52-338d56165df5",
         "cluster_name": "custom cluster name 2",
         "last_checked_at": "2020-11-07T08:32:37.690Z",
         "meta": {
           "cluster_version": "4.18.12"
-        }
+        },
+        "impacted": ""
```

Comment:
I think the server will not respond with empty string but with the field missing

Reply:
You are true, they marked it as optional, then this key can be possibly omitted. I'll rewrite the code so that is also considers missing "impacted"

Reply:
@ikerreyes, updated, PTAL